pytest_fixtures/component/provision_gce.py

```diff
@@ -108,6 +117,16 @@ def session_puppet_default_partition_table(session_puppet_enabled_sat):
     return ptable
 
 
+@pytest.fixture(scope='session')
+def session_default_partition_table(session_target_sat):
+    ptable = (
+        session_target_sat.api.PartitionTable()
+        .search(query={'search': f'name="{DEFAULT_PTABLE}"'})[0]
+        .read()
+    )
+    return ptable
```

Comment:
You can get rid of this one we have https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/component/partition_table.py#L6
```suggestion
```